qlib/tests/__init__.py

```diff
@@ -51,3 +51,32 @@ def setUpClass(cls, enable_1d_type="simple", enable_1min=False) -> None:
             dataset_cache=None,
             **cls._setup_kwargs,
         )
+
+
+class TestOperatorData(TestAutoData):
+    @classmethod
+    def setUpClass(cls, enable_1d_type="simple", enable_1min=False) -> None:
+        from qlib.data.filter import NameDFilter
```

Comment:
Why do you import them in the method instead of at the head of this file?

Reply:
Put import to the head of file is good to me. 

I mimic from here.
https://github.com/microsoft/qlib/blob/19689024d49e61e1187990d2c5f35b6e45ec513c/qlib/backtest/__init__.py#L174
Why would you guys import in the method in above case.
Thanks.

Reply:
That is for avoiding recursive import.
Both of the two modules import each other.

Reply:
Got it.